exasol_integration_test_docker_environment/lib/base/task_dependency.py

```diff
@@ -9,23 +9,15 @@ def __init__(self, id: str, representation: str):
         self.representation = representation
         self.id = id
 
-    def to_json(self):
-        jsonpickle.set_preferred_backend('simplejson')
-        jsonpickle.set_encoder_options('simplejson', sort_keys=True)
-        return jsonpickle.encode(self)
+    @property
+    def formatted_representation(self):
+        assert '"' not in self.representation
```

Comment:
This is causing problems with the DockerBuildTask => it uses luigi.ListParameter(), and str() of that contains '"'